src/config/dynamic-theme-fixes.config

```diff
@@ -2072,6 +2072,25 @@ td.c {
 
 ================================
 
+localhost:88*
```

Comment:
localhost should not do the trick as you can also acces it trough your local ip `127.0.0.1` or some even host them extern for GPU power.

Reply:
So what address should I use? `http://127.0.0.1:88*`?

Reply:
I will look for a better way